### PR TITLE
refactor: spelling fixes

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/dripip/commit/633d79020e7adb311487e8127d8bb434917aa355#commitcomment-47150911

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/dripip/commit/5502d801a698e3663f79dc0bc82313ea3486be5c

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: src/cli/commands/preview.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export class Preview extends Command {
     trunk: flags.string({
       default: '',
       description:
-        'State which branch is trunk. Defaults to honuring the "base" branch setting in the GitHub repo settings.',
+        'State which branch is trunk. Defaults to honouring the "base" branch setting in the GitHub repo settings.',
```
Comment (jsoref) [inline src/cli/commands/preview.ts:10]: I tend to favor `honor` over `honour`, but I think this repo favours the extra `u`.

File: src/lib/changelog/data.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * This module deals with building changelogs from serieses.
+ * This module deals with building changelogs from series.
```
Comment (jsoref) [inline src/lib/changelog/data.ts:2]: `series` is its own plural.

You can say `a series of series` if you mean that, or `multiple series` if you think `series` is ambiguous.

File: src/lib/git.ts [hunk 3]
```diff
@@ -225,7 +225,7 @@ export async function checkSyncStatus(git: Simple): Promise<SyncStatus> {
     : 'diverged'
 }
 
-export interface BasicGithubRepoInfo {
+export interface BasicGitHubRepoInfo {
```
Comment (jsoref) [inline src/lib/git.ts:228]: I'm not sure how public your API is.

It's fairly trivial for me to split out changes like this, or to drop them entirely.

Mostly, the goal is to fix spelling of the brand when written standalone as `GitHub`. Just as one shouldn't write `priSMa laBS`.

File: src/lib/github-ci-environment.ts [hunk 4]
```diff
@@ -19,17 +19,17 @@ export interface GithubCIEnvironment {
...
 /**
- * Parse the Github CI Environment. Returns null if parsing fails which should
- * mean it is not a Github CI Environment.
+ * Parse the GitHub CI Environment. Returns null if parsing fails which should
+ * mean it is not a GitHub CI Environment.
  *
  * @remarks
  *
- * Github docs: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
+ * GitHub docs: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
```
Comment (jsoref) [inline src/lib/github-ci-environment.ts:27]: Here's an example where the spelling should be fixed.

File: src/lib/package-json.ts [hunk 5]
```diff
@@ -26,15 +26,15 @@ type ValidatedPackageJson = PackageJson & { name: string }
  */
 function validate(pj: PackageJson | undefined): ValidatedPackageJson {
   if (!pj) {
-    throw new Error('Could not find pacakge.json')
+    throw new Error('Could not find package.json')
```
Comment (jsoref) [inline src/lib/package-json.ts:29]: This class of bugs is frustrating to end user devs.

File: src/utils/context-checkers.ts [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import { validateNPMAuthSetup } from '../lib/npm-auth'
-import { Validator } from './contrext-guard'
+import { Validator } from './context-guard'
```
Comment (jsoref) [inline src/utils/context-checkers.ts:2]: The file is renamed in the next bit (I can't easily link to the rename)

**Review by jsoref — COMMENTED** [review]
I can split things up or drop things, just let me know.

File: src/cli/commands/preview.ts (hunk 1 above)
Comment (jasonkuhrt) [reply src/cli/commands/preview.ts:10]: It might be more OSS friendly to go US Spelling here though.

File: src/lib/changelog/data.ts (hunk 2 above)
Comment (jasonkuhrt) [reply src/lib/changelog/data.ts:2]: > You can say a series of series if you mean that, or multiple series if you think series is ambiguous.

Yeah that would be great

File: src/lib/git.ts (hunk 3 above)
Comment (jasonkuhrt) [reply src/lib/git.ts:228]: This fine fine!

File: src/cli/commands/preview.ts (hunk 1 above)
Comment (jsoref) [reply src/cli/commands/preview.ts:10]: If you're up for it, I'm happy to make the change. I generally make the smallest change possible (my PRs are already quite large, so I try not to make waves).

Comment (jasonkuhrt) [reply src/cli/commands/preview.ts:10]: Yeah let's do it :)

**Pull request merged**